mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -491,13 +491,13 @@ fn get_locked_amount_by_other_swaps(ctx: &MmArc, except_uuid: &Uuid, coin: &str)
         })
 }
 
-pub fn active_swaps_using_coin(ctx: &MmArc, coin: &str) -> Result<Vec<Uuid>, String> {
+pub fn active_swaps_using_coins(ctx: &MmArc, coins: &[String]) -> Result<Vec<Uuid>, String> {
```

Comment:
First of all, https://github.com/KomodoPlatform/atomicDEX-API/pull/1525#discussion_r1032402184
So I said here https://github.com/KomodoPlatform/atomicDEX-API/pull/1525#discussion_r1033815798, this algo has `O(swaps.len())` complexity if only `coins: &HashSet<String>`. Same for `get_matching_orders`